content/rule_book/quiz_process.md

```diff
@@ -1,106 +1,173 @@
 # Quiz Process
 
+**Quiz Question**
+: Every component of a question within the context of a quiz including all components of a written question along with quiz context such as question number
+
+**Number**
+: The full identification of which question a given question is within a quiz, which can be purely a numeric or an alphanumeric
+
+**Numeric**
+: A question number with only numbers such as 17
+
+**Alphanumeric**
+: A question number with numbers and letters such as 17A
+
+**Read Question**
+: Everything the quizmaster reads from the quiz question
+
+**Answer**
+: Everything the quizzer says while answering a question after being recognized by the quizmaster until their answer duration has elapsed or being called correct or incorrect
```

Comment:
change "answer duration" to "question answer duration period" to be consistent

Reply:
444e5ce